adminer/Dockerfile

```diff
@@ -1,3 +1,2 @@
 #ddev-generated
-FROM adminer:standalone
-ADD ddev-login.php /var/www/html/plugins-enabled/
+FROM shyim/adminerevo:latest
```

Comment:
This file serves no purpose here (that I can understand). It should just be removed, and the docker-compose.adminer.yaml should use the upstream image.